Resources/Windows/dlls/angle.ru.txt

```diff
@@ -1,42 +1,42 @@
-Руководство по ANGLE в ONScripter-RU
-Использование DirectX 9/DirectX 11 рендереров.
-
-Должен быть установлен эпизод 5 или новее (onscripter-ru r2846 или новее).
-
-
-Установка:
-1. Отредактируйте файл ons.cfg (или default.cfg если ons.cfg отсутствует) и
-добавьте следующие строчки при необходимости.
-
-2. Включите рендерер (ОБЯЗАТЕЛЬНО):
-prefer-renderer=ANGLE2 (или ANGLE3 для DirectX 11)
-
-3. Укажите режим рендеринга (ТОЛЬКО ДЛЯ LEGACY БИБЛИОТЕК):
-render-self=yes
-
-4. Укажите предпочтительный компилятор шейдеров (ПРИ ПРОБЛЕМЕ ЗАПУСКА):
-d3dcompiler=d3dcompiler_43.dll
-
-
-Примечания:
-Для использования legacy библиотек необходимо скопировать содержимое папки
-dlls\legacy в соответствующие папки с заменой:
-dlls\6.1  для Windows 7 и новее
-dlls\5.1  для Windows XP и новее
-
-Использование библиотек Legacy требует render-self=yes, но обычно
-снижает требования к оперативной памяти. Тем не менее, стоит выбирать
-максимально производительный вариант, например, используя отображение
-FPS в названии окна. Включается строчкой:
-show-fps
-
-Для более подробного логирования ошибок используйте консоль:
-use-console
-
-Возможно указать собственную библиотеку d3dcompiler, если она
-установлена.
-
-Возможно использование более новых версий libGLESv2.dll и libEGL.dll,
-для этого скопируйте их, например, из папки веб браузера Opera или
-Mozilla Firefox. Убедитесь, что данные библиотеки имеют 32-битную
-разрядность.
+Руководство по ANGLE в ONScripter-RU
```

Comment:
Did you change line endings here? If you did, please revert, because windows users still need CR LF.